drivers/firmware/scmi/nxp/Kconfig

```diff
@@ -6,3 +6,14 @@ config ARM_SCMI_NXP_VENDOR_EXTENSIONS
 	select CRC
 	help
 	  When enabled, additional SCMI features specific to NXP will be available
+
+if ARM_SCMI_NXP_VENDOR_EXTENSIONS
+
+config NXP_SCMI_CPU_DOMAIN_HELPERS
```

Comment:
I think we need to get rid of this "*_HELPERS" naming scheme. It just doesn't make sense anymore for cases such as this one where there's no subsystem API....

anyways, I'd say leave as-is until we find a solution.

Reply:
So we could keep it this time and fix later